src/autoCollection/metrics/httpMetricsInstrumentation.ts

```diff
@@ -51,20 +65,28 @@ export class HttpMetricsInstrumentation extends InstrumentationBase {
     return [this._getHttpDefinition(), this._getHttpsDefinition()];
   }
 
-  private _htppRequestDone(metric: IHttpMetric) {
+  private _httpRequestDone(metric: IHttpStandardMetric) {
```

Comment:
Curious as to why it is named `IHttpStandardMetric` and not `ISpanStandardMetric`? We will probably need this for other standard metric signals from different types like db and messaging.